### PR TITLE
bevm: simplify contract registration

### LINKED ISSUE #2206: Simplify BEVM contracts
Change the way BEVM registers its contracts according to this:

https://github.com/c4dt/byzcoin/blob/master/pkg.base/cothority/bevm/service.go

### PR DESCRIPTION
**Pull request by cgrigis**
**What this PR does**

This PR simplifies the way the `bevm` contract is registered.

Closes #2206.

---

🙅‍ Friendly checklist:

- [X] 0. Code comments are added (or updated) when/where needed and explain the WHY of the code.
- [X] 1. Design choices, user documentation and any additional doc are added (or updated) in READMEs.
- [X] 2. Any new behaviour is tested and small units of code that can be are unit tested.
- [X] 3. Code comments are added on tests to explain what they do.
- [X] 4. Errors are systematically wrapped with a meaningful message using `xerrors.Errorf` and the `%v` verb.
- [X] 5. Hard limit of 80 chars is always respected.
- [X] 6. Changes are backward compatible.
- [X] 7. Indentation level does not exceed 5, although 4 is already suspicious.
- [X] 8. Functions, files, and packages are kept to a manageable size and decomposed into smaller units if needed.
- [X] 9. There are no magic values.

### CODE REVIEW

File: bevm/service.go [hunk 1]
```diff
@@ -13,43 +11,9 @@ func init() {
...
-	if err != nil {
-		return nil, xerrors.Errorf("failed to register BEvmValue "+
-			"contract: %v", err)
-	}
-
-	return s, nil
+	// Register contracts
+	log.ErrFatal(byzcoin.RegisterGlobalContract(ContractBEvmID,
+		contractBEvmFromBytes))
+	log.ErrFatal(byzcoin.RegisterGlobalContract(ContractBEvmValueID,
```
Comment (ineiti) [inline bevm/service.go]: Is this needed at all?

Comment (ineiti) [reply bevm/service.go]: I mean the registration of `ContractBEvmValueID`

Comment (cgrigis) [reply bevm/service.go]: Yes, see my comment on #2206...

**Comment by ineiti** [comment]
If you move this into R4M it can be merged ;)

**Comment by cgrigis** [comment]
> If you move this into R4M it can be merged ;)

Thank you. :) I was just waiting for the green from Travis ;)

**Pull request merged**